sdk/data/aztables/README.md

```diff
@@ -1,151 +1,348 @@
+handle(err)
+
+filter := "PartitionKey eq 'markers' or RowKey eq 'id-001'"
+options := &ListEntitiesOptions{
+    Filter: &filter,
+    Select: to.StringPtr("RowKey,Value,Product,Available"),
+    Top: to.Int32Ptr(15),
+}
+
+pager := client.List(options) // pass in "nil" if you want to list all entities
+for pager.NextPage(context.Background()) {
+    resp := pager.PageResponse()
+    fmt.Printf("Received: %v entities\n", len(resp.Entities))
+
+    for _, entity := range resp.Entities {
+        var myEntity aztables.EDMEntity
+        err = json.Unmarshal(entity, &myEntity)
+        handle(err)
+
+        fmt.Printf("Received: %v, %v, %v, %v\n", myEntity.Properties["RowKey"], myEntity.Properties["Value"], myEntity.Properties["Product"], myEntity.Properties["Available"])
+    }
+}
+
+err := pager.Err()
+handle(err)
+```
 
-- `TableServiceClient` - Client that provides methods to interact at the Table Service level such as creating, listing, and deleting tables
-- `TableClient` - Client that provides methods to interact at an table entity level such as creating, querying, and deleting entities within a table.
-- `Table` - Tables store data as collections of entities.
-- `Entity` - Entities are similar to rows. An entity has a primary key and a set of properties. A property is a name value pair, similar to a column.
-
-Common uses of the Table service include:
-
-- Storing TBs of structured data capable of serving web scale applications
-- Storing datasets that don't require complex joins, foreign keys, or stored procedures and can be de-normalized for fast access
-- Quickly querying data using a clustered index
-### Create the Table service client
-
-First, we need to construct a `TableServiceClient`.
-
-### Create an Azure table
-Next, we can create a new table.
-
-
-### Get an Azure table
-The set of existing Azure tables can be queries using an OData filter.
-
-
-### Delete an Azure table
-
-Individual tables can be deleted from the service.
-
-
-### Create the Table client
-
-To interact with table entities, we must first construct a `TableClient`.
-
+## Troubleshooting
 
-### Add table entities
+### Error Handling
 
-Let's define a new `TableEntity` so that we can add it to the table.
+All I/O operations will return an `error` that can be investigated to discover more information about the error. In addition, you can investigate the raw response of any response object:
+```golang
+resp, err := client.CreateTable(context.Background(), nil)
+if err != nil {
+    err = errors.As(err, azcore.HTTPResponse)
+    // handle err ...
+}
+```
 
-Using the `TableClient` we can now add our new entity to the table.
+### Logging
 
+This module uses the classification based logging implementation in azcore. To turn on logging set `AZURE_SDK_GO_LOGGING` to `all`. If you only want to include logs for `aztables`, you must create your own logger and set the log classification as `LogCredential`.
```

Comment:
Joel & I renamed "classification" to "event" (I think).

Reply:
I don't see any reference to 'event' in azcore